spec/models/work_spec.rb

```diff
@@ -457,26 +475,6 @@
     end
   end
 
-  # Is this test providing any coverage?
```

Comment:
This method was removed because we are not relying on active storage any more...  We have not for a long time